File: app/src/main/java/com/momen/babyactivities/IntroActivity.java
package com.momen.babyactivities;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class IntroActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_intro);
    }

    public void Goals(View view) {
        startActivity(new Intent(this, DisplayActivity.class)
                .putExtra("index", 0));
    }

    public void Info(View view) {
        startActivity(new Intent(this, DisplayActivity.class)
                .putExtra("index", 1));
    }

    public void Conponants(View view) {
        startActivity(new Intent(this, ChooseLevelActivity.class));
    }
}

File: app/src/main/java/com/momen/babyactivities/IntroActivity.kt
package com.momen.babyactivities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.momen.babyactivities.R
import android.content.Intent
import android.view.View
import android.view.WindowManager
import com.momen.babyactivities.DisplayActivity
import com.momen.babyactivities.ChooseLevelActivity
import kotlinx.android.synthetic.main.activity_intro.*
import kotlin.system.exitProcess

class IntroActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_intro)
        // getWindow().addFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN);

        backBtn.setOnClickListener { onBackPressed() }
        exitBtn.setOnClickListener {this.finishAffinity()}
    }

    fun Goals(view: View?) {
        startActivity(
            Intent(this, DisplayActivity::class.java)
                .putExtra("index", 0)
        )
    }

    fun Info(view: View?) {
        startActivity(
            Intent(this, DisplayActivity::class.java)
                .putExtra("index", 1)
        )
    }

    fun Conponants(view: View?) {
        startActivity(Intent(this, ChooseLevelActivity::class.java))
    }
}